content/blog/2022/czi-global-communities-proposal/index.md

```diff
@@ -0,0 +1,62 @@
+---
+title: "Open grant narrative: A Collaborative Interactive Computing Service Model for Global Communities"
+subtitle: ""
+summary: ""
+authors: ["Chris Holdgraf"]
+tags: []
+categories: [partnerships]
+date: "2022-08-28"
+featured: false
+draft: false
+---
+
+
+We recently submitted a grant to [Chan Zuckerberg Initiative](https://chanzuckerberg.com/) and wish to share some details about it as well as the grant narrative for others to read and re-use.
+
+{{< cta cta_text="Go to Zenodo record" cta_link="https://zenodo.org/record/7025288" cta_new_tab="true" >}}
+
+Read on for a quick overview of the proposal.
+
+## Collaborators
+
+This grant is a collaboration between several leading organizations in open infrastructure, community, and global leadership:
+
+- [2i2c](https://2i2c.org)
+- [The Carpentries](https://carpentries.org/about/)
+- [Center for Scientific Collaboration and Community Engagement](https://www.cscce.org/)
+- [Invest in Open Infrastructure](https://investinopen.org/)
+- [MetaDocencia](https://www.metadocencia.org/)
+- [Open Life Science](https://openlifesci.org/)
+
+## Problem statement
+
+Cloud infrastructure is a powerful way to broaden access to workflows and infrastructure across the globe.
+However, it is also inaccessible to many for a variety of reasons:
+
+- There is a large, diverse, and messy ecosystem of open source tools to facilitate cloud infrastructure.
+- Most communities don't already have skills in utilizing cloud workflows.
+- Running infrastructure in the cloud takes dedicated time and expertise that many communities lack.
+- Many communities do not have organized communities of practice around cloud infrastructure.
+
+These issues are true for most scientific communities, but they are exacerbated in countries that are historically under-represented in the global scientific community.
```

Comment:
These issues are true for most scientific communities, but they are exacerbated in countries that are often marginalized in the global scientific community.

Reply:
Under-represented is softwer, but puts the onus on the weakest. marginalized puts the onus on the strongest and the folks that have the power to fix the problem :-)

Reply:
thanks for this guidance, I will use your wording